src/django_components/component.py

```diff
@@ -836,19 +833,18 @@ class MyComp(Component):
         self._types = args_type, kwargs_type, slots_type, data_type, js_data_type, css_data_type
         return self._types
 
-    def _validate_inputs(self) -> None:
-
+    def _validate_inputs(self, args: Tuple, kwargs: Any, slots: Any) -> None:
```

Comment:
Passed in the values of args, kwargs, & slots, directly into the function, so it can be called before we populate `Component.input`.